dolt/routers.py

```diff
@@ -39,14 +39,14 @@ def db_for_write(self, model, **hints):
         if not is_global_router_enabled():
             return None
 
-        if is_versioned_model(model):
-            return None
-
         if is_dolt_model(model):
```

Comment:
Double check if this is still needed

Reply:
@andrew-wm-arthur This doesn't seem to make a difference with PRs being written to main now but I think it's the correct change. Wdyt?